packages/components/data-table-manager/src/column-settings-manager/column-settings-manager.tsx

```diff
@@ -101,7 +101,7 @@ export const handleColumnsUpdate = (
       : selectedColumns;
 
     const columns = isSwap ? selectedColumns : availableColumns;
-    const draggedColumn = columns.find(
+    const draggedColumn = columns!.find(
```

Comment:
Sorry, but if this is optionally don't we conditionally chain here?

Reply:
we know that `availableColumns` will always be `[]` because of the default param on line 134 - optional chaining is probably the better choice though